core/server/apps/default-cards/cards/markdown.js

```diff
@@ -1,14 +1,17 @@
 var SimpleDom   = require('simple-dom'),
     tokenizer   = require('simple-html-tokenizer').tokenize,
-    Showdown    = require('showdown-ghost'),
-    converter   = new Showdown.converter({extensions: ['ghostgfm', 'footnotes', 'highlight']}),
+    markdownConverter  = require('../../../utils/markdown-converter'),
     parser;
 
 module.exports = {
         name: 'card-markdown',
         type: 'dom',
         render(opts) {
             parser = new SimpleDom.HTMLParser(tokenizer, opts.env.dom, SimpleDom.voidMap);
-            return parser.parse('<div class="kg-card-markdown">' + converter.makeHtml(opts.payload.markdown || '') + '</div>');
+            return parser.parse(''
+                + '<div class="kg-card-markdown">'
```

Comment:
I think so, we're still using the mobiledoc format and hopefully our koenig editor will be coming back in some form later on. If we get rid of the card classes now then there will be another breaking change post 1.0 when we expand mobiledoc support again.